server/src/main/java/com/linecorp/centraldogma/server/DefaultConfigValueConverter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.server;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+enum DefaultConfigValueConverter implements ConfigValueConverter {
+    INSTANCE;
+
+    private static final String FILE = "file";
+
+    // TODO(minwoox): Add more prefixes such as classpath, url, etc.
+    private static final List<String> SUPPORTED_PREFIX = ImmutableList.of(FILE);
```

Comment:
Should we also support `plain` in case a value starts with `file` or some prefixes which are actually part of the data, e.g. `plain:file:;!abc`? Although it would rarely happen.